pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/SchemasResource.java

```diff
@@ -183,6 +218,34 @@ private static void handleGetSchemaResponse(AsyncResponse response,
 
     }
 
+    private static void handleGetAllSchemasResponse(AsyncResponse response,
+                                                    List<SchemaAndMetadata> schemas, Throwable error) {
+        if (isNull(error)) {
+            if (isNull(schemas)) {
+                response.resume(Response.status(Response.Status.NOT_FOUND).build());
+            } else if (schemas.size() == 0) {
+                response.resume(Response.status(Response.Status.NOT_FOUND).build());
+            } else {
+                response.resume(
+                        Response.ok()
+                                .encoding(MediaType.APPLICATION_JSON)
+                                .entity(GetAllVersionsSchemaResponse.builder().getSchemaResponses(
+                                        schemas.stream().map(schemaAndMetadata -> GetSchemaResponse.builder()
```

Comment:
Can you extract the `schemaAndMetadata` to `GetSchemaResponse` transformation into a common method? I think it should be reused across this class.